_includes/subscribe-modal.html

```diff
@@ -0,0 +1,44 @@
+<div class="modal fade" id="subscribe-success" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered subscribe-modal " role="document">
+        <div class="modal-content text-center">
+            <div class="modal-header">
+                <button type="button" class="close" onclick="window.history.pushState({path: window.location.pathname },'',window.location.pathname)"  data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+                <img src="/assets/img/guides/img-subscribe.png" alt="subscribe_image" /><br />
+            </div>
+            
+            <div class="modal-body">
+                <h2 class="no-anchor">Subscribe to Gruntwork</h2>
+                <ul>
+                    <li>
+                        <h3 class="modal-title no-anchor">Gruntwork Service Catalog</h3>
+                        <p>Get access to over 300,000 lines of reusable, battle-tested, production-grade infrastructure code for AWS and GCP.
+                            <a href="#">Coming Soon</a>
```

Comment:
Coming soon?

Reply:
Shouldn't this be a "Learn More" link that goes to the IaC Lib page?

Reply:
Ooh I thought this was going to be linked with the service catalog currently being worked on. So Sam suggested we change that to coming soon. I'll fix that now